2024Project-main/2024FinalYearProject/Data/RepositoryBase.cs

```diff
@@ -44,5 +48,10 @@ public async Task UpdateAsync(T? entity)
             await _context.SaveChangesAsync();
 
         }
+
+        public async Task SaveAsync()
```

Comment:
Remove this unnecessary method , SaveAsync already handled on AddAsync and UpdateAsync (see line 48)